src/java/org/apache/cassandra/index/sai/plan/StorageAttachedIndexQueryPlan.java

```diff
@@ -37,7 +37,7 @@
 
 public class StorageAttachedIndexQueryPlan implements Index.QueryPlan
 {
-    private static final boolean USE_MULTI_RANGE_READ_COMMAND = Boolean.parseBoolean(System.getProperty("cassandra.sai.use_multi_range_read_command", "false"));
+    private static final boolean USE_MULTI_RANGE_READ_COMMAND = Boolean.parseBoolean(System.getProperty("cassandra.sai.use_multi_range_read_command", "true"));
```

Comment:
Is this still useful if the fix works?  If we keep it probably should leave a comment saying what it is and why it's here, eg, the jiras or something.

Reply:
No I don't think it is now. I've removed it and set SAI to always use the `MultiRangeReadCommand`